proto/zepben/protobuf/hc/LogMessage.proto

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright 2023 Zeppelin Bend Pty Ltd
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/.
+ */
+
+syntax = "proto3";
+option java_multiple_files = true;
+option java_package = "com.zepben.protobuf.hc";
+option csharp_namespace = "Zepben.Protobuf.HC";
+package zepben.protobuf.hc;
+
+import "google/protobuf/timestamp.proto";
+import "google/protobuf/struct.proto";
+
+message LogMessage {
+  /**
+   * The ID of the work package.
+   */
+  string workPackageId = 1;
+
+  /**
+   * The name of the scenario.
+   */
+  string scenario = 2;
+
+  /**
+   * The year in the scenario.
+   */
+  int32 year = 3;
+
+  /**
+   * The name of the feeder.
+   */
+  string feeder = 4;
```

Comment:
Question: Should these be nullable like in the logs table? This would make sense for handling cases where a failure is not specific to a scenario/year/feeder

Reply:
sorry missed this earlier - strings will just be empty strings which we can treat as null. the int however will be 0 if unset, so i think it's better to make it nullable.

Reply:
Used `google.protobuf.NullValue` for year.